src/polygon.jl

```diff
@@ -0,0 +1,69 @@
+"""
+    getlevel(dataset::ArchGDAL.AbstractDataset, reqlevel::Int) 
+Get layer of the desired level from the dataset
+"""
+function getlevel(dataset::ArchGDAL.AbstractDataset, reqlevel::Int)
```

Comment:
I would remove the type annotation, or at least write it as dataset::AbstractDataset instead.